.github/CODEOWNERS

```diff
@@ -73,6 +73,8 @@
 /libs/application/template-api-modules/src/lib/modules/templates/driving-school-confirmation/       @island-is/juni
 /libs/application/templates/operating-license/                                                      @island-is/juni
 /libs/application/template-api-modules/src/lib/modules/templates/operating-license/                 @island-is/juni
+/libs/application/templates/driving-license-duplicate/                                              @island-is/juni
```

Comment:
Isn't there a better english name for this. It sounds like it is a copy paste of the driving-license application 😆

Reply:
It does sound a bit confusing for sure, I haven't found a better translation anywhere :/ Lögreglan uses "apply for a copy .." in their translations so duplicate is where I landed. If you have any other suggestions I would love to hear them!

And I would like to get the application out feature flagged in this release so we can get stakeholders to test it and also so that I can take updates on the data providers in this application into consideration when updating to the new data providers set up for all our applications. Going to attempt to structure all the ökunáms applications api actions into a smoother ecosystem with the update.